nb_conda/handlers.py

```diff
@@ -259,7 +259,7 @@ def get(self):
 
 # there is almost no text that is invalid, but no hyphens up front, please
 # neither all these suspicious but valid caracthers...
-_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)$"
+_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)"
```

Comment:
This is unfortunate, because now we'll match anything that starts out with a legal name but has forbidden characters later on - unless the JS implementation of regex does something other than the python one.